src/lib/y2storage/autoinst_proposal.rb

```diff
@@ -190,23 +191,24 @@ def root?(devices)
       issues_list.add(:missing_root)
     end
 
-    # Finds a suitable devicegraph using the guided proposal approach
+    # Finds a suitable devicegraph using the initial guided proposal approach
     #
-    # If the :desired target fails, it will retry with the :min target.
-    #
-    # @param devicegraph [Devicegraph]       Starting point
-    # @param drives      [AutoinstDrivesMap] Devices map from an AutoYaST profile
-    # @return [Devicegraph] Proposed devicegraph using the guided proposal approach
+    # @see Y2Storage::GuidedProposal.initial
     #
     # @raise [Error] No suitable devicegraph was found
-    # @see proposed_guided_devicegraph
+    #
+    # @param devicegraph [Devicegraph] Starting point
+    # @param drives [AutoinstDrivesMap] Devices map from an AutoYaST profile
+    #
+    # @return [Devicegraph] Proposed devicegraph using the guided proposal approach
     def propose_guided_devicegraph(devicegraph, drives)
       devicegraph = clean_graph(devicegraph, drives, [])
-      begin
-        guided_devicegraph_for_target(devicegraph, drives, :desired)
-      rescue Error
-        guided_devicegraph_for_target(devicegraph, drives, :min)
-      end
+
```

Comment:
np: Why those blank lines?

Reply:
Only a matter of styling. I tend to separate lines that are no a direct consequence of the previous one (more or less).

Reply:
Well, in that case, 209 and 211 looks quite related to me :)